debian/changelog

```diff
@@ -1,3 +1,58 @@
+snapcraft (2.27) xenial; urgency=medium
+
+  [ Sergio Schvezov ]
+  * core: switch to using rpath for clasic confinement. (#1094)
+  * python plugin: do the right thing with classic. (#1093)
+  * meta: support for the environment keyword. (#1103)
+  * meta: correct the deprecation for `setup/gui` use. (#1107)
+  * tests: avoid snapping progress to leak into report. (#1114)
+  * core: setup core to support classic confinement. (#1112)
+  * cleanbuild: allow talking to a remote. (#1121)
+  * New upstream release 2.27 (LP: #1663229)
+
+  [ Kyle Fazzari ]
+  * schema,copy plugin: better errors when item has no value. (#1096)
+  * cleanbuild: include snap directory in tarball. (#1102)
+  * repo: remove symlinks to libc. (#1100)
+  * catkin plugin: don't pass args to setup.sh. (#1099)
+  * pluginhandler: support more complex stage-packages. (#1059)
+  * repo: cache stage packages for faster future use. (#1122)
+
+  [ Leo Arias ]
+  * tests: do not rely on project_dir. (#1078)
+  * lifecycle: print the command needed to clean the dirty part (#1097)
+  * tests: check the CLA on all the commits. (#1106)
+  * tests: rename the integration test snaps. (#1105)
+  * misc: consistently use a dash for copyright years. (#1101)
+  * tests: rename the waf test snap. (#1111)
+  * ci: fix the env vars passed to the docker container in travis. (#1125)
+
+  [ Marco Trevisan ]
+  * meta: properly get the icon extension from splitted name. (#1092)
+  * tests: fix or ignore PEP8 static errors. (#1085)
+
+  [ John Lenton ]
+  * ux: print the version on startup when running with --debug (#1086)
+
+  [ Dan Chapman ]
+  * qbs plugin: add plugin support for the Qt Build Suite (qbs). (#1079)
+
+  [ Bayard Randel ]
+  * store: implement delta uploads in push. (#940)
+
+  [ Joe Talbott ]
+  * store: expose store errors to the cli. (#1110)
+  * Make git pulls less error prone due to history changes. (#1052)
```

Comment:
Agh, I did, my apologies. I've successfully formed a commit habit for this, but not when hitting the merge button apparently.